setup/podman.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+if [ -z "$SUDO_USER" ]; then
+    echo "This script is intended to be run as sudo. Exiting"
+    exit 1
+fi
+
+UBUNTU_MAJOR_VERS=$(lsb_release -rs | cut -c-2)
+
+add_sources () {
+    if [[ "$UBUNTU_MAJOR_VERS" -ge "23" ]]; then
+        # appropriate podman versions are already available
+        return
+    fi
+
+    if [ ! -f /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list ]; then
+        mkdir -p /etc/apt/keyrings
+        curl -fsSL "https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/Release.key" \
+          | gpg --dearmor \
+          | tee /etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg > /dev/null
+        # NOTE: opensuse seems to be having https difficulties, so switching to http temporarily
+        echo \
+          "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg]\
+            http://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/ /" \
+          | tee /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list > /dev/null
+        apt-get update -qq
+    fi
+}
+
+#add_sources
+
+# Clean previous podman installations, just in case
+apt remove -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+#rm -f /etc/systemd/system/docker.socket
+#rm -f /etc/systemd/system/sockets.target.wants/docker.socket
+#rm -f /etc/systemd/system/multi-user.target.wants/docker.service
+
+# Also might want to take care of /var/lib/docker (containers and images still stored here)
+
+#rm -f /etc/systemd/system/podman.service
+#rm -f /etc/systemd/system/podman-restart.service
+#rm -f /etc/systemd/system/podman-auto-update.service
+#rm -f /etc/systemd/system/podman-auto-update.timer
+
+#rm -f /etc/systemd/user/podman.service
+#rm -f /etc/systemd/user/default.target.wants/podman.service
+#rm -f /etc/systemd/user/sockets.target.wants/podman.socket
+#rm -f /etc/xdg/systemd/user/podman.socket
+#systemctl  --user daemon-reload
+
+#Also look in /usr/lib/systemd/user/podman.socket
+# /usr/lib/systemd/user/podman.service
```

Comment:
Are these comments needed? I know some of this was necessary for cleaning up after a podman 3 install, but it's not clear how these comments are useful here without additional comments to explain how and when they should be used.

Reply:
Yeah, this is a 'working' PR, but I haven't cleaned everything up yet. It's rough around the edges. I don't believe these comments serve a purpose